crates/swift-bridge-cli/src/clap_exec.rs

```diff
@@ -0,0 +1,41 @@
+use std::collections::HashMap;
+use std::path::{Path};
+use clap::ArgMatches;
+use swift_bridge_build::{ApplePlatform, create_package, CreatePackageConfig};
+
+/// Executes the correct function depending on the cli input
+pub fn handle_matches(matches: ArgMatches) {
+    match matches.subcommand_name() {
+        Some("create-package") => handle_create_package(matches.subcommand_matches("create-package").unwrap()),
+        _ => unreachable!("No subcommand or unknown subcommand given") // Shouldn't happen
+    }
+}
+
+/// Executes the `create-package` command
+fn handle_create_package(matches: &ArgMatches) {
+    let bridges_dir = matches.value_of("bridges-dir").unwrap(); // required
```

Comment:
If know that if we use the `#[derive(Debug, Parser)]` clap macro we should automatically get a nice help message if someone forgot a required field.

Is there an easy way to do that without using the macro?

If not, should we just use the macro? What do you think?

Reply:
Using the Builder api has the same result, so this unwrap statement should never fail as Clap will always show a helpful eerror message.